templates/jaasai/experts/omnivector.html

```diff
@@ -0,0 +1,171 @@
+{% extends "_layout.html" %}
+
+{% block title %}Juju expert - Omnivector{% endblock %}
+{% block description %}Omnivector is committed to delivering the highest quality hardware, software, and data services{% endblock %}
+
+{% block content %}
+<div class="p-strip--image is-light u-no-padding--bottom" style="background-image:url('https://assets.ubuntu.com/v1/c883127d-download.svg')">
+  <div class="row">
+    <div class="col-8">
+      <span class="expert__strap">Juju Expert Partners</span>
+      <h1>Scalable, Cloud Agnostic Software and Data Services</h1>
+      <h2 class="p-heading--four">Omnivector is committed to delivering the highest quality hardware, software, and data services. Using industry best practices and agile delivery frameworks, we ensure that our solutions are modern, scalable and maintainable.</h2>
```

Comment:
"Using the industries best practices and agile delivery frameworks we ensure that our solutions are modern, scalable and maintainable."